client/src/types/index.ts

```diff
@@ -110,3 +110,8 @@ export interface KaKaoRegion {
   x: number;
   y: number;
 }
+
+export interface awsUploadInfo {
```

Comment:
타입 이름은 맨 앞글자가 대문자가 되는 게 좋을 것 같아요!
```suggestion
export interface AwsUploadInfo {
```

Reply:
넵!